content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -233,23 +233,22 @@ When a Pod's containers are Ready but at least one custom condition is missing o
 
 ## Container probes
 
-A [Probe](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#probe-v1-core) is a diagnostic
-performed periodically by the
+A {{< api-reference page="workloads-resources/container" anchor="Probe" >}}
```

Comment:
Is there are reason we use 'pod-v1' to refer to a Pod definition, but we are not using 'container-v1' to refer to a Container definition?